### ISSUE TITLE
Minor bug: React needs DOM target

### ISSUE THREAD

**Issue by harryli0088** [frozeman/meteor-build-client#61]
Hello! First I want to say that your package is awesome and has helped me a lot!

I encountered a minor bug when building for React. React needs its target to exist in the DOM. For example in my ```client/main.html``` file, I have this in the body:
```html
<body>
    <div id="react-target"></div>
</body>
```

I added the ```react-target``` element to the build's ```index.html``` file and everything works fine.
FYI I'm running Meteor 1.8.1 and React 16.5.0

Thanks again!

**Comment by dr-dimitru** [comment]
Hello @harryli0088 ,

This issue is addressed in [`v1.0`](https://github.com/frozeman/meteor-build-client/tree/v1.0) suggesting using `static-html` pacjage to define default template structure, in your case all you need is single file `body.html` with content like:

```html
<!-- /client/body.html -->
<body>
    <div id="react-target"></div>
</body>
```

I encourage you to try [`v1.0`](https://github.com/frozeman/meteor-build-client/tree/v1.0) and report your experience.
Please see updated docs on [branch `v1.0`](https://github.com/frozeman/meteor-build-client/tree/v1.0)

**Comment by dr-dimitru** [comment]
Hello @harryli0088 ,

This issue addressed in [`v1.0.0`](https://github.com/frozeman/meteor-build-client/releases/tag/1.0.0), and published last night to [NPM](http://npmjs.com/package/meteor-build-client). Please, give it a try and let me know if there's any issues.
